### ISSUE TITLE
Display experimental data  first

### ISSUE THREAD

**Issue by ValWood** [geneontology/amigo#549]
In AmiGO it would be really nice if experimental data was shown first,  followed by inferences:

http://amigo.geneontology.org/amigo/term/GO:0140274

It often looks as though everything is IEA and the interesting experimental stuff is hidden- we should be presenting and promoting our curated experimental data primarily.

**Comment by ValWood** [comment]
I don't think so. Lots of species only have IEA data, and so you would see nothing by default (and there is nothing wrong with IEA data if there is nothing else available).


If it isn't possible to  promote the experimental data to the top of the list, is it possible to make the data from the key annotated species appear first?  This would help, and it would be  furhter improved by the redundant annotation filtering  proposed  by @cmungall 
(filtering alone won't help because most species have no EXP annotation)

All the EXP data here is pombe but it is right at the bottom...... it doesn't really promote  GO curation in the best light. If a newbie comes to AmiGO (or anyone) they surely they should see the experimental data/key species  first.

**Comment by cmungall** [comment]
Agreed.

Another option is prioritizing most specific terms, this would tend to correlate with experimental I expect (but results may vary), and would have the side-effect of always listing a redundant after more specific terms

**Comment by ukemi** [comment]
I think we need to re-conceptualize the visualization. I would vote for using the structure of the ontology as the key to visualization rather than evidence. An interactive graphical display would be ideal.

**Comment by ValWood** [comment]
> Another option is prioritizing most specific terms

this wouldn't help if you are viewing a  specific term though as in the example:
http://amigo.geneontology.org/amigo/term/GO:0140274


>I think we need to re-conceptualize the visualization. I would vote for using the structure of the ontology as the key to visualization rather than evidence. An interactive graphical display would be ideal.

this would be nice.

It would be useful to know more about entry points/landing pages. Do most people come in via gene, or term?
Are they mainly direct from a Google search, or links to GO terms from other resources  or AmiGO searches ?
